composer.json

```diff
@@ -14,6 +14,7 @@
         "friendsofphp/php-cs-fixer": "^2.16",
         "infection/infection": "^0.15.0",
         "phan/phan": "^2.4",
+        "php-coveralls/php-coveralls": "^2.2",
```

Comment:
What is it used for?

Reply:
For coverage reporting and status messages. If not enabled by default, these can be [enabled in the settings](https://coveralls.io/github/infection/include-interceptor). Basically that's what codecov does for the main repo, but [with pretty graphs](https://coveralls.io/repos). As I like it much more than codecov, I think we may give this alternative service a try.

Reply:
Thanks, I didn't noticed that we already have coverage uploading script:

https://github.com/infection/include-interceptor/blob/master/.travis.yml#L46

Reply:
...These were the lines I didn't remove when I added my stock Makefile.